docs/tailwind.config.cjs

```diff
@@ -11,7 +11,20 @@ module.exports = {
     "'./node_modules/@svelteness/kit-docs/client/**/*.svelte'"
   ],
 
-  plugins: [...config.plugins, overrides]
+  plugins: [...config.plugins, overrides],
+  theme: {
+    extend: {
+      colors: {
+        conf: {
+          99: '#5aea98',
+          95: '#5dea5a',
+          90: '#bcea5a',
+          80: '#ffe600',
+          70: '#eab05a'
+        }
+      }
+    }
+  }
```

Comment:
@Adamj1232 What are these colors for?

Reply:
Those colors are for the gradient on the gas page

Reply:
Can we scope them down to just the gas component? If we're only using them there, I'd rather not add them to global theme.

Reply:
@gesquinca they are pretty specific variables and would be required to be he to use the tailwinds implementation of gradients. This pattern follows what we are doing in other projects(Gas extensions). I tagged you in the implementation.

Reply:
All cleaned up!